packages/ia-zendesk-help-widget/package.json

```diff
@@ -28,7 +28,7 @@
     "storybook:build": "build-storybook -o _site -s storybook-static"
   },
   "dependencies": {
-    "@internetarchive/lazy-loader-service": "^0.1.0",
+    "@internetarchive/lazy-loader-service": "^0.2.0",
```

Comment:
This is required for the Offshoot pages, but I'm unsure whether this upgrade will cause issues with the widget on pages that aren't in Offshoot yet. Do those pages still require the older version?

Reply:
Update: shouldn't be an issue -- petabox will continue using the older widget version.